ConsoleApp9/Piece/BishopPiece.cs

```diff
@@ -2,14 +2,92 @@ namespace Chess;
 
 public class BishopPiece : AbstractPiece
 {
-    public BishopPiece(string symbol, PieceColor color, (int, int) position, GameState gameState) : base(symbol, color, position, gameState) { }
+    private bool _isCaptured = false;
```

Comment:
Note to myself: I overrode all the AbstractPiece methods in this class, likely didn't mean to do so in the end I figured abstractPiece/some abstraction was needed, it might as well be AbstractPiece